lib/puppet/functions/compliance_markup/telemetry.rb

```diff
@@ -20,22 +20,16 @@ def telemetry(key)
     end
     retval
   end
+
   def codebase()
     'compliance_markup::telemetry'
   end
+
```

Comment:
Is `environment()` used?

Reply:
This doesn't refactor the helpers at this time.